rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java

```diff
@@ -65,6 +70,35 @@ public static TableMetaCache getInstance() {
         return tableMetaCache;
     }
 
+    @Override
+    protected String getCacheKey(DataSourceProxy dataSourceProxy, String tableName) {
+        StringBuilder cacheKey = new StringBuilder(dataSourceProxy.getResourceId());
+        cacheKey.append(".");
+        String defaultTableName = tableName.replace("`", "");
+
+        DatabaseMetaData databaseMetaData = null;
+        try {
+            databaseMetaData = dataSourceProxy.getPlainConnection().getMetaData();
+        } catch (SQLException e) {
+            LOGGER.error("Could not get connection, use default cache key", e.getMessage(), e);
+            return cacheKey.append(cacheKey.append(defaultTableName)).toString();
+        }
+
+        try {
+            //prevent duplicated cache key
+            if (databaseMetaData.supportsMixedCaseIdentifiers()) {
+                cacheKey.append(defaultTableName);
+            } else {
+                cacheKey.append(defaultTableName.toLowerCase());
+            }
+        } catch (SQLException e) {
+            LOGGER.error("Could not get supportsMixedCaseIdentifiers in connection metadata, use default cache key", e.getMessage(), e);
+            return cacheKey.append(cacheKey.append(defaultTableName)).toString();
+        }
+
+        return cacheKey.toString();
```

Comment:
We can return cacheKey.append(defaultTableName).toString() as default.

Reply:
Thanks. Fixed.

Reply:
If change, the code would like this
```
StringBuilder cacheKey = new StringBuilder(dataSourceProxy.getResourceId());
        cacheKey.append(".");
        String defaultTableName = tableName.replace("`", "");

        DatabaseMetaData databaseMetaData = null;
        try {
            databaseMetaData = dataSourceProxy.getPlainConnection().getMetaData();
        } catch (SQLException e) {
            LOGGER.error("Could not get connection, use default cache key", e.getMessage(), e);
        }

        try {
            //prevent duplicated cache key
            if (databaseMetaData!= null && !databaseMetaData.supportsMixedCaseIdentifiers()) {
                return cacheKey.append(defaultTableName.toLowerCase()).toString();
            }
        } catch (SQLException e) {
            LOGGER.error("Could not get supportsMixedCaseIdentifiers in connection metadata, use default cache key", e.getMessage(), e);
        }

        return cacheKey.append(defaultTableName).toString();
```

Reply:
OK.